lsl/data/lwana-ssmif.txt

```diff
@@ -499,144 +499,272 @@ FEE_PORT[64] 8
 N_RPD 128
 
 # RPD_ID[] will be set to "UNK" (no identification) for any cables #'s <= N_RPD not identified
-# *** RPD_ID[cable#] goes here:
-# FIXME JD: CT to provide this
+# *** RPD_ID[cable#] <label> goes here:
+
+RPD_ID[1]  Stand 1-XX
```

Comment:
Is the parser ok with spaces in the labels?

Reply:
Looks like it as far as LSL is concerned:  https://github.com/lwa-project/lsl/blob/lwana/lsl/common/stations.py#L1051

Not sure about MCS.

Reply:
I think MCS should be ok with it as well:  https://github.com/lwa-project/monitor_and_control/blob/lwana/tp/tprs.c#L94

Reply:
Changed to exclude spaces inside the labels